sway-fmt-v2/src/utils/language/expr/asm_block.rs

```diff
@@ -1,8 +1,8 @@
 use crate::{
-    formatter::*,
+    formatter::{shape::LineStyle, *},
```

Comment:
`*` :triggered:
oof, it's done in other places too

Reply:
What's wrong here?